docs/contributing/developing.rst

```diff
@@ -51,7 +45,20 @@ Compile the assets:
 
     $ npm run build
 
-Any Wagtail sites you start up in this virtualenv will now run against this development instance of Wagtail.  We recommend using the `Wagtail Bakery demo site <https://github.com/wagtail/bakerydemo/>`_ as a basis for developing Wagtail. Keep in mind that the setup steps for a Wagtail site may include installing a release version of Wagtail, which will override the development version you've just set up. In this case, you should install the site before running the ``pip install -e`` step, or re-run that step after the site is installed.
+You will then need to link your local Wagtail codebase to a pre-existing project you may have using Wagtail.
+
+Then, with your preferred `virtual enviornment <https://docs.python.org/3/tutorial/venv.html>`_ activated, install the Wagtail package in development mode with the included testing and documentation dependencies as a dependency in your project (testing or documentation dependencies could also be considered optional depending on your needs):
```

Comment:
Typo here (and in other places) - enviornment -> environment. Also, we only need to include the link on the first mention of virtual environments. (We started by saying "you should be familiar with them", after all!)